app/src/NativeBase/Components/SegmentedPicker.tsx

```diff
@@ -44,15 +45,25 @@ const SegmentedPicker: FC<SegmentedPickerProps> = ({
         <Pressable flexGrow="1" onPress={option.onPress}>
           <Box
             alignItems="center"
-            backgroundColor={option.isSelected ? 'white' : 'transparent'}
+            _dark={{
+              backgroundColor: option.isSelected ? '#636366' : 'transparent',
```

Comment:
Please can you put all of the new colour values into StaTheme.tsx and reference them using colour names -- that's the NativeBase theming approach, plus it's more helpful to see a colour name, anyone can more quickly see what it is.  

As a rule we want to keep all colour definitions in the theme file, and these colours may well need to be used in other components in the future, so defining them there makes that possible.

Please see https://github.com/Scottish-Tech-Army/Volunteer-app/blob/main/APP_DEVELOPMENT.md#theming--approach

Reply:
Will sort this tomorrow, thanks! Is there a clever way to deal with the transparent hex values in the divider, or should it just be a 8 digit hex value in the theme?

Reply:
Great, thanks. Nothing different to do for semi transparent colours that I know of. I’m assuming there definitely needs to be transparency set on those? (they can’t just be normal solid colours?)

Reply:
(A solid background colour would probably give slightly better performance)

Reply:
[This is now done I think](https://github.com/Scottish-Tech-Army/Volunteer-app/pull/158/commits/bc10609558b14e28d4f72de0b38956a38a69be36). I made them solid colours.